ath.sh

```diff
@@ -8,9 +8,12 @@ cd "$(dirname "$0")"
 # https://github.com/jenkinsci/acceptance-test-harness/releases
 export ATH_VERSION=5497.vca_4a_876045ce
 
-# TODO use Artifactory proxy?
+MVN='mvn -B -ntp -Pquick-build -am -pl war package'
+if [[ -n ${MAVEN_SETTINGS-} ]]; then
```

Comment:
```suggestion
if [[ -v MAVEN_SETTINGS ]]; then
```
?

Reply:
TIL about `-v`, testing the existence of the corresponding variable.

I've copied this test [from the BOM](https://github.com/jenkinsci/bom/blob/fe602c266c057636a42a8cbd3a2b77043d717849/prep.sh#L5-L10), I don't mind changing it but I'm not sure what would be the benefit of checking the existence of the variable instead of checking if the variable value length is greater than 0.

Any input to add?

Reply:
Either is fine, this was just in case you wanted something easier to read. In the `bom` case it is from https://github.com/jenkinsci/bom/pull/1149 which I guess was for local testing.

Reply:
spoiler: It's nitpick of course :)
I have a personal preference for `-n` for the following reasons:
- portability with macOS (and other shells)
- Coherency of the syntax: `-n` is homogeneous with numerous other operators with the `$` . The `-v` feels weird as it only reference the variable for its name.

Reply:
Thanks for the additional input, I'll keep the most portable current version with `-n`.